2023-participants/interested-technical-writers.md

```diff
@@ -48,3 +48,4 @@ _NOTE: PRs will be auto-closed until the beginning of the 2023 Season of Docs pr
 > Rejoice Anodo | <ul><li>[Twitter](https://twitter.com/IsaacRejoice2) </li> <li> [GitHub](https://github.com/iribama) </li> <li> [Dev.to](https://dev.to/rejoice) </li> <li>[Portfolio](https://linktr.ee/rejoiceanodo)</li> </ul> | **Hello!** I am a cloud/DevOps engineer and technical writer with experience in creating, maintaining and implementing infrastructure solutions and servers. I also have unique oral and written skills, with experience in creating technical documentation and written content. </br> <br> **Technical Writing Interests:** <ul><li>Blog posts</li><liArticles></li> <li>Tutorials</li><li>API Documentation</li><li>User guides</li><li>Long form texts</li><li>General technical content</li></ul> **Languages, Frameworks and SKills:** <ul><li>Configuration management: Linux, Ansible</li><li>Cloud Platforms: AWS, GCP</li><li>Open-Source: Git, GitHub, OSS</li><li>Containerization: Docker, Kubernetes</li><li>HTML and CSS</li><li>Infrastructure as Code (IaC): Terraform</li><li>Scripting: Python, Bash</li></ul> **Writing tools:** <ul><li>Markdown</li><li>Notion</li><li>GitBook</li><li>Static site generators</li><li>Use of AI tools</li></ul>
 > Nuruddeen Sambo | [LinkedIn](https://www.linkedin.com/in/nuruddeen-umar-sambo/), [Portfolio](https://sambonuruddeen.github.io/portfolio/) | Software Development, Machine Learning, Open Source, Technical Writing
 > Oyinkansola Awosan | <ul><li>[LinkedIn](http://www.linkedin.com/in/oyinawosan/) </li> <li> [Portfolio](https://linktr.ee/OyinkansolaAwosan) </li></ul> | **Hello there!** 😀 I am Oyinkansola, a data scientist, machine learning engineer, site reliability engineer and technical writer. I am constantly honing my skills and I look forward to making valuable contributions to this year's Google Season of Docs</br> **Interests:** <ul><li>API Documentation</li><li>User Manuals and Guides</li><li>Long-form Articles</li></ul> **Tech Skills:** <ul><li>HTML, CSS, JavaScript</li><li>Node/ExpressJS</li><li>Python</li><li>TypeScript</li><li>Git and GitHub</li></ul>  **Writing tools:** <li>Markdown</li></ul>  **Others:**<ul><li>Postman</li> <li>Notion</li> <li>Slack</li><li>Jira</li></ul>
+> Were Samson Bruno | [Linkedin](https://www.linkedin.com/in/were-samson/), [Portfolio Website](https://www.weresamson.com) | Tech Blogger, Front-end Software engineer, JavaScript, Open Source Contributor
```

Comment:
Adedayo Ojaomo |  <ul><li>[LinkedIn](https://linkedin.com/in/adedayoojaomo)</li><li>[Twitter](https://https://twitter.com/Kvng_Victor_) </li> <li> [GitHub](https://github.com/Vojaomo27) </li></li> </ul>  **Hello!** I am a cloud engineer, technical writer, and data scientist with experience in creating, and implementing infrastructure solutions. I have good oral and written skills, with experience in creating great technical written content. I am looking forward to making contributions to this year's Google season of docs </br> <br> ** My Technical Writing Interests are:** <ul><li>Blog posts</li><li>Articles></li> <li>API Documentation</li><li>Tutorials</li></ul> **Languages and Frameworks:** <ul><li>Cloud Platforms: Azure, GCP</li><li>Open-Source: Git, GitHub, OSS</li><li>Containerization: Docker, Kubernetes</li><li>HTML and CSS</li><li>Infrastructure as Code (IaC)</li><li>Scripting: Python</li></ul>